assets/js/modules/ads/components/settings/SettingsView.stories.js

```diff
@@ -108,3 +109,59 @@ PaxConnected.decorators = [
 		);
 	},
 ];
+
+export const IceEnabled = Template.bind( null );
+IceEnabled.storyName = 'With ICE enabled';
+IceEnabled.scenario = {
+	label: 'Modules/Ads/Settings/SettingsView/ICE/Enabled',
+	delay: 250,
```

Comment:
Do all of your story scenarios definitely need a delay to run successfully in VRTs? My understanding is that this is most often not needed and slows down our VRTs overall. These could be removed throughout the SettingsView stories for both the Ads and Analytics modules IMO.

Reply:
Thanks @benbowler I was merely going on what the default stories had, and assumed there was a reason. I can remove and we can see what happens.